config/stp.py

```diff
@@ -391,13 +614,20 @@ def stp_global_forward_delay(_db, forward_delay):
     ctx = click.get_current_context()
     db = _db.cfgdb
     check_if_global_stp_enabled(db, ctx)
-    is_valid_forward_delay(ctx, forward_delay)
-    is_valid_stp_global_parameters(ctx, db, "forward_delay", forward_delay)
-    update_stp_vlan_parameter(ctx, db, "forward_delay", forward_delay)
-    db.mod_entry('STP', "GLOBAL", {'forward_delay': forward_delay})
+    current_mode = get_global_stp_mode(db)
+    if current_mode == "pvst":
+        is_valid_forward_delay(ctx, forward_delay)
+        is_valid_stp_global_parameters(ctx, db, "forward_delay", forward_delay)
+        update_stp_vlan_parameter(ctx, db, "forward_delay", forward_delay)
+        db.mod_entry('STP', "GLOBAL", {'forward_delay': forward_delay})
+    elif current_mode == "mst":
+        db.mod_entry('STP_MST', "GLOBAL", {'forward_delay': forward_delay})
+        update_mst_instance_parameters(ctx, db, 'forward_delay', forward_delay)
```

Comment:
We dnt need this call update_mst_instance_parameters

Reply:
I have addressed this comment, please review